### PR TITLE
added discrod,twitch,figma,leetcode,spotify,appleMusic svg and updated the package to latest versions

### PR DESCRIPTION
**Pull request by babyo77**
fixes- https://github.com/kingsmen732/portfolio/issues/4  https://github.com/kingsmen732/portfolio/issues/1

### CODE REVIEW

**Comment by kingsmen732** [comment]
Hey, I admire your work here kindly add an image with the updated images instead of the old one so it stands as reference for your work.

**Comment by babyo77** [comment]
[
Screenshot 2024-10-06 at 10 59 22 PM
](url)

File: src/assets/spotify.svg
Comment (kingsmen732) [inline src/assets/spotify.svg:1]: image
fix the image size with respect to other aspects in mind

File: src/assets/appleMusic.svg
Comment (kingsmen732) [inline src/assets/appleMusic.svg:1]: image
fix the image size with respect to other aspects in mind, this SVG doesn't fit the entire button.

**Review by kingsmen732 — CHANGES_REQUESTED** [review]
kindly review the 2 issues in this PR

**Comment by babyo77** [comment]
make it rounded
Screenshot 2024-10-07 at 1 35 04 AM
 3xl

**Comment by kingsmen732** [comment]
we are planning on a stable feature changing the roundness of the button solves the issue temporarily with misalignment and no consistency in the design.
image

**Comment by kingsmen732** [comment]
we love your work and effort , I just want to make it more presentable work of your contribution @babyo77

**Pull request closed without merging**